cli/pkg/kctrl/cmd/core/validate_columns.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package core
```

Comment:
Maybe the file name should be `print_table.go`?

Reply:
fixed it.